ajo/personalization-server-side/README.md

```diff
@@ -0,0 +1,210 @@
+# Server-side Personalization
+
+## Overview
+
+This sample demonstrates using Adobe Experience Platform to get personalization content from Adobe Target. The web page changes based on the personalization content returned.
```

Comment:
we should replace Adobe Target with Adobe Journey Optimizer in this file

Reply:
Done